frontend/packages/metal3-plugin/src/selectors/index.ts

```diff
@@ -3,8 +3,8 @@ import * as _ from 'lodash';
 import { K8sResourceKind, MachineKind } from '@console/internal/module/k8s';
 import { getName } from '@console/shared';
 
-export const getOperationalStatus = (host) => _.get(host, 'status.operationalStatus');
-export const getProvisioningState = (host) => _.get(host, 'status.provisioning.state');
+export const getHostOperationalStatus = (host) => _.get(host, 'status.operationalStatus');
```

Comment:
Not new, but it would be better to add types to all of these helper functions

Reply:
Done, I'll create BaremetalHostKind for host in a separate PR.